### PR TITLE
Admin dashboard mb

### PR DESCRIPTION
**Pull request by Maxwell-Baird**
Created the admin dashboard that allows the admin to see users and orders that are happening. 
Allows the admin to change status of orders that are packaged to shipped
Created the show page for admin/users/:id

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@ def create
       elsif user.role == 2
         redirect_to '/merchant/dashboard'
       elsif user.role == 3
-        redirect_to '/admin/dashboard'
+        redirect_to '/admin'
```
Comment (colintalex) [inline app/controllers/sessions_controller.rb:28]: Glad we got this changed

File: app/controllers/admin/dashboard_controller.rb [hunk 2]
```diff
@@ -2,4 +2,15 @@ class Admin::DashboardController < Admin::BaseController
...
   end
+
+  def user
+    @user = User.find(params[:id])
+  end
+
+  def update
+    order = Order.find (params[:id])
+    order.status = "Shipped"
+    order.save
```
Comment (DavidTTran) [inline app/controllers/admin/dashboard_controller.rb:13]: I think I did this a couple of times as well, surprised there's no way to both update and save

File: app/views/admin/dashboard/show.html.erb [hunk 3]
```diff
@@ -1 +1,35 @@
 <h1> Welcome Admin </h1>
+<h3> Orders </h3>
+<h4> Packaged </h4>
+<article id="Packaged">
+  <% Order.where("status = 'Packaged'").each do |order| %>
```
Comment (colintalex) [inline app/views/admin/dashboard/show.html.erb:5]: I feel this logic should be inside a controller, thoughts?

File: config/routes.rb [hunk 4]
```diff
@@ -55,7 +55,9 @@
   end
 
   namespace :admin do
-    get '/dashboard', to: "dashboard#show"
+    get '/', to: "dashboard#show"
```
Comment (colintalex) [inline config/routes.rb:58]: Thumbs up!

File: app/models/order.rb [hunk 5]
```diff
@@ -9,6 +9,10 @@ def grandtotal
     item_orders.sum('price * quantity')
   end
 
+  def find_user
+    User.find(self.user_id).name
+  end
```
Comment (DavidTTran) [inline app/models/order.rb:14]: This looks very weird..

File: spec/models/order_spec.rb [hunk 6]
```diff
@@ -33,6 +33,11 @@
     it 'grandtotal' do
       expect(@order_1.grandtotal).to eq(230)
     end
+
+    it 'find_user' do
+      expect(@order_1.find_user).to eq("David")
```
Comment (colintalex) [inline spec/models/order_spec.rb:38]: Nice method

File: app/models/order.rb (hunk 5 above)
Comment (Maxwell-Baird) [reply app/models/order.rb:14]: I can rename it to find_user_name

**Review by colintalex — APPROVED** [review]
Looks good!

File: config/routes.rb (hunk 4 above)
Comment (DavidTTran) [inline config/routes.rb:58]: I was gonna ask if we we're gonna make those minor changes, but looks like you already did it 👍

Comment (DavidTTran) [reply config/routes.rb:58]: Gotta do the same for merchant as well

**Pull request merged**